### DISCUSSION TITLE
IPTV dont work after Update to 2.4.7 - There must be at least 1 Vif as upstream.

### DISCUSSION THREAD

**Discussion by WW1983** [fabianishere/udm-iptv#241] [Q&A] [answered]
### Diagnostic Information

=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15
LAN Interfaces: br0  br5
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
31: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 87.178.208.96 peer 62.155.246.153/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.246.153 proto kernel scope link src 87.178.208.96
=== Service Logs ===
,Mar 08 08:07:36 Dream-Machine-Pro systemd[1]: udm-iptv.service: Scheduled restart job, restart counter is at 14                                     8.
Mar 08 08:07:36 Dream-Machine-Pro systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Mar 08 08:07:36 Dream-Machine-Pro systemd[1]: udm-iptv.service: Consumed 81ms CPU time.
Mar 08 08:07:36 Dream-Machine-Pro systemd[1]: Started IPTV support for the UniFi Dream Machine.
Mar 08 08:07:36 Dream-Machine-Pro udm-iptvd[25918]: NATing IPTV network ranges (if necessary)
Mar 08 08:07:36 Dream-Machine-Pro udm-iptvd[25918]: Creating static routes (if necessary)
Mar 08 08:07:36 Dream-Machine-Pro udm-iptvd[25918]: Monitoring interface ppp0 for changes
Mar 08 08:07:36 Dream-Machine-Pro udm-iptvd[25918]: Setting up IGMP Proxy
Mar 08 08:07:36 Dream-Machine-Pro udm-iptvd[25918]: Using igmpproxy...
Mar 08 08:07:36 Dream-Machine-Pro udm-iptvd[25918]: Starting IGMP Proxy

### Describe the Bug

Hello,

I use the UDM-Pro with Magenta TV. It was running fine until the update. After the update I reinstalled udm-iptv and added my configuration. Unfortunately I get the following error message:

Mar 08 08:07:31 Dream Machine Pro igmpproxy[25730]: There must be at least 1 Vif as upstream.

### Expected Behavior

bugfix

**Comment by fabianishere** [comment, marked as answer]
Looking at the diagnostics, it seems that the `igmpproxy` started by udm-iptv is running.

The error you are getting is from the `igmpproxy` service, which you should disable:

```
systemctl disable --now igmpproxy
```

**Reply by WW1983** [reply]
After disable ipmpproxy and restart the service, the error seems to be gone.

Thanks.
